src/messages/manifestjson.js

```diff
@@ -555,3 +555,12 @@ export function permissionFirefoxAndroidUnsupportedByMinVersion(
     file: MANIFEST_JSON,
   };
 }
+
+export const RESTRICTED_HOMEPAGE_URL = {
+  code: 'RESTRICTED_HOMEPAGE_URL',
```

Comment:
The new rule code has to also added to the markdown table in docs/rules.md (like you did also for the new rule you created in #3425).